deployment/src/strongmind_deployment/rails.py

```diff
@@ -82,10 +84,38 @@ def ecs(self):
         self.kwargs['env_vars'] = self.env_vars
         self.kwargs['container_image'] = container_image
 
-        self.container = ContainerComponent("container",
-                                            pulumi.ResourceOptions(parent=self),
-                                            **self.kwargs
-                                            )
+        web_entry_point = self.kwargs.get('web_entry_point', ["sh", "-c",
```

Comment:
I'm pretty anxious about determining the entry point here, rather than in a Dockerfile. This won't work for Canvas later.

Canvas's prep steps take so long that they fail ECS out. We'll want those to be part of the CD before we start deploying the app.

Reply:
Can we set this to just use the standard container entry point, (which is I think what we've been doing already?) and have the difference be for the worker only?

Reply:
These are just default entry points if none are passed in. My intent is to allow app devs to set the entry points or leave them blank if they prefer.